scripts/branding.ts

```
@@ -0,0 +1,85 @@
+  }
+  return parsed;
+};
+
+export const loadBranding = (requestedBrandId = process.env.BRAND?.trim() || DEFAULT_BRAND_ID): BrandingResolution => {
+  if (!BRAND_ID_PATTERN.test(requestedBrandId)) {
+    throw new Error(
+      `[branding] Invalid BRAND value "${requestedBrandId}". Only alphanumeric characters, hyphens, and underscores are allowed.`,
+    );
+  }
+
+  const defaultBranding = readBrandingFile(defaultBrandingPath);
```

ESLint

[error] Remove this useless assignment to variable "defaultBranding".

(sonarjs/no-dead-store)